data/data/bootstrap/files/usr/local/bin/installer-masters-gather.sh

```diff
@@ -54,5 +57,15 @@ mkdir -p "${ARTIFACTS}/rpm-ostree"
 sudo rpm-ostree status >& "${ARTIFACTS}/rpm-ostree/status"
 sudo rpm-ostree ex history >& "${ARTIFACTS}/rpm-ostree/history"
 
+# Collect system information specific to IBM Linux Z (s390x) systems. The dbginfo
+# script is available by default as part of the s390-utils rpm package
+if [ "$arch" == "s390x" ]
+then
+    echo "Gathering dbginfo for the s390x system"
+    mkdir -p "${ARTIFACTS}/node-dbginfo"
+    /usr/sbin/dbginfo.sh -d "${ARTIFACTS}/node-dbginfo"
```

Comment:
What is the difference between gathering the dgbinfo on the bootstrap machine and the master machine that the former needs to use `sudo` while the latter does not?